src/main/java/mate/academy/Main.java

```diff
@@ -2,11 +2,14 @@
 
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 public class Main {
+    private static final Integer THREAD_POOL_SIZE = 4;
+
     public static void main(String[] args) {
         // Feel free to play with AsyncRequestProcessor in this main method if you want
-        ExecutorService executor = null; // Provide implementation that fits your needs
+        ExecutorService executor = Executors.newFixedThreadPool(THREAD_POOL_SIZE);
```

Comment:
Consider using a try-with-resources statement or adding a shutdown hook to ensure that the executor is properly shutdown in case of an unexpected error or runtime shutdown. This helps to avoid potential resource leaks.